evento/src/views/EventPage/index.js

```diff
@@ -36,20 +54,46 @@ class EventPage extends Component {
 		});
 	}
 
+	onAttendButtonClick() {
+		const eventId = this.props.match.params.eventId;
+
+		fetch(`/events/${eventId}/attendees`, {
+			method: 'POST',
+			headers: { 'Authorization': Cookie.get('auth_token') }
+		})
+		.then(response => {
+			if (!response.ok) return Promise.reject();
+			// If successiful attending, re-fetch attendees
+			this.fetchAttendees();
+		})
+		.catch(() => this.setState({ errorMessage: "Something went wrong" }));
+	}
+
+	isUserAttending() {
+		let user = Cookie.get('user');
+		return JSON.stringify(this.state.attendees).includes(user);
```

Comment:
So 
```js
const userCookie = Cookie.get('user');
if(!userCookie) return false;
const = JSON.parse(userCookie);
return this.state.attendees.some(attendee => attendee.id == user.id);
```
vs
```js
const user = Cookie.get('user');
return JSON.stringify(this.state.attendees).includes(user);
```
?

I think the latter is clearer. It just literally says that "attendees includes user" and that's what the function is doing.